CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # 1.0.0 TBD
 
+## Fixes
+- Remove unsupported curl options [11](https://github.com/bugsnag/bugsnag-cli/pull/11)
+- Fix using port within endpoint url CLI option [10](https://github.com/bugsnag/bugsnag-cli/pull/10)
```

Comment:
The tool isn't released yet, so we don't need to start tracking changes right now. When we tag the first release, we just need an initial entry that summarises the initial functionality.